config/locales/devise.en.yml

```diff
@@ -78,7 +78,7 @@ en:
         subject: 'Mastodon: Security key authentication enabled'
         title: Security keys enabled
     omniauth_callbacks:
-      failure: Could not authenticate you from %{kind} because "%{reason}".
+      failure: Could not authenticate you from %{kind} because “%{reason}”.
```

Comment:
This line doesn't look related to this change, other than that - LGTM

Reply:
As you can see I made a commit to change the two occurrences of straight quotes in translation:

Use curved quotations marks instead of straight ones
https://github.com/tootsuite/mastodon/pull/16129/commits/6336ca771d360694f7a6725cfd286d51529bc97f

Now it’s unrelated but it’s a very small change and uncontroversial I believe, since curved quotes are also used everywhere on https://docs.joinmastodon.org/.

Reply:
Oh, didn't notice that this was multiple commits, it's fine then